Simplenote/SimplenoteAppDelegate+Swift.swift

```diff
@@ -454,8 +457,37 @@ extension SimplenoteAppDelegate: NSMenuItemValidation {
     func validateToogleMarkdownPreviewItem(_ item: NSMenuItem) -> Bool {
         noteEditorViewController.validateToogleMarkdownPreviewItem(item)
     }
+
+    /// Updates `active` state of top view controllers based on the current first responder
+    ///
+    func updateActivePanel(with responder: NSResponder) {
+        let viewControllers: [NSResponder] = [tagListViewController, noteListViewController, noteEditorViewController]
+        var responder = responder
+        while true {
```

Comment:
Suggestion (to drop the `while true`)

```
        let viewControllers: [NSResponder] = [tagListViewController, noteListViewController, noteEditorViewController]
        var nextResponder: NSResponder? = responder

        while let currentResponder = nextResponder {
            if viewControllers.contains(currentResponder) {
                tagListViewController.isActive = tagListViewController == nextResponder
                noteListViewController.isActive = noteListViewController == nextResponder
                break
            }

            nextResponder = currentResponder.nextResponder
        }
```

Okay maybe not worth it 😂 

`while true` always makes me freak out a bit (but all of the scenarios are handled so it's all good!

Reply:
Good suggestion, less code :-)